IPython/core/display.py

```diff
@@ -858,8 +876,10 @@ def __init__(self, data=None, url=None, filename=None, format=None,
             if ext is not None:
                 if ext == u'jpg' or ext == u'jpeg':
                     format = self._FMT_JPEG
-                if ext == u'png':
+                elif ext == u'png':
                     format = self._FMT_PNG
+                elif ext == u'bmp' or ext == u'bm':
```

Comment:
Is `.bm` really an extension for bitmaps? I've never seen it, Wikipedia [doesn't mention it](https://en.wikipedia.org/wiki/BMP_file_format), and my system does not recognise it. Let's stick to `.bmp` unless there's a clear need for another extension.